cypress/README.md

```diff
@@ -1,25 +1,44 @@
 # Cypress tests and our approach to E2E testing
 
+[Introduction to Cypress](https://docs.cypress.io/guides/core-concepts/introduction-to-cypress.html#Cypress-Is-Simple)
+
 This folder contains all the simorgh application end-to-end (E2E) tests and exclusively uses cypress as a testing framework. At present we run all E2E tests on all builds and deploys to all environments including live.
 
 We aim to keep the code reusable but also to leverage the cypress framework as much as possible, here are some helpful hints. NB we're not perfect we're breaking our own guidelines and need help tidying up and making our tests more scalable.
 
-## Helpful hints
+## Directory Structure
+
+[About cypress directory structure](https://docs.cypress.io/guides/core-concepts/writing-and-organizing-tests.html#Folder-Structure)
+
+<!-- prettier-ignore -->
+    .
+    ├── fixtures                    # Cypress pre-defined
+    ├── integration                 # Cypress tests
+    ├── plugins                     # Internal behavior of Cypress
+    └── support                     # Helper utilities for tests
+        └── config*
+
+\*Config exists in one place. Config for running tests is in /support/config/ and config for expected results should either be hardcoded in a test or imported from application config (for example service translations).
+
+## Running Test
+
+[Running Cypress locally](https://github.com/bbc/simorgh#end-to-end-tests)
+
+Use the scripts from Simorgh's [package.json](https://github.com/bbc/simorgh/blob/latest/package.json#L39#L40) to run Cypress test locally on LIVE/TEST environment.
+
+## Best Practises
+
+[Cypress best practices](https://docs.cypress.io/guides/references/best-practices.html)
+[Cypress defaults commands](https://docs.cypress.io/api/api/table-of-contents.html)
 
-- Use the default cypress commands e.g. cy.get().
+- Use the custom commands in `/support/command.js` directory.
```

Comment:
Should be that we use the default commands and the custom commands, we don't want a custom command created that is superficially different to the deafult. e.g. the getElement helper currently in /support.